### DISCUSSION TITLE
do you recommend this PPA for ubuntu 22.04.4 ??

### DISCUSSION THREAD

**Discussion by Zorono** [realmazharhussain/gdm-settings#257] [Q&A]
Do you recommend this PPA for Ubuntu 22.04.4 ?? **ubuntuhandbook1/gdm-settings** (i found it in https://ubuntuhandbook.org/index.php/2023/10/install-gdm-settings-ubuntu-22-04/)

**Comment by realmazharhussain** [comment]
I don't
1. I have not tested it myself
2. It ships v4.1 which is old at this point (there have been several bug fixes after 4.1)

But if it works for you, then by all means, use it.
